CHANGELOG.md

```diff
@@ -2,31 +2,27 @@
 
 ## Upcoming Release
 
-This upcoming release of the agent adds instrumentation for Active Support caching operations, Action Mailer operations, and Action Mailbox operations.
+The upcoming release of the agent introduces additional Ruby on Rails instrumentation (especially for Rails 6 and 7) for various Action*/Active* libraries whose actions produce Active Support notifications events.
```

Comment:
I think this may do something to the markdown you're not expecting. 

You could try: 
Action\*/Active\*

To escape the *'s

Reply:
Addressed with [e4f9e0d](https://github.com/newrelic/newrelic-ruby-agent/pull/1745/commits/e4f9e0d6dcd952490c9345daa8471c4b55ae637e), thank you!